falco/values.yaml

```diff
@@ -15,6 +15,11 @@ containerd:
   enabled: true
   socket: /run/containerd/containerd.sock
 
+kubernetesSupport:
+  enabled: true
+  certFile: /var/run/secrets/kubernetes.io/serviceaccount/token
```

Comment:
I went with "certFile" as thats what its mostly referred to as here
```
 -K <bt_file> | <cert_file>:<key_file[#password]>[:<ca_cert_file>], --k8s-api-cert <bt_file> | <cert_file>:<key_file[#password]>[:<ca_cert_file>]
```

Reply:
I agree with you that those do not feel great.

I'd propose a very generic `apiAuth` instead.

Also documenting the full format of the field with a comment could help users:
```
<bt_file> | <cert_file>:<key_file[#password]>[:<ca_cert_file>], --k8s-api-cert <bt_file> | <cert_file>:<key_file[#password]>[:<ca_cert_file>]
                               Use the provided files names to authenticate user and (optionally) verify the K8S API server identity.
                               Each entry must specify full (absolute, or relative to the current directory) path to the respective file.
                               Private key password is optional (needed only if key is password protected).
                               CA certificate is optional. For all files, only PEM file format is supported. 
                               Specifying CA certificate only is obsoleted - when single entry is provided 
                               for this option, it will be interpreted as the name of a file containing bearer token.
                               Note that the format of this command-line option prohibits use of files whose names contain
                               ':' or '#' characters in the file name.
```

wdyt?